### DISCUSSION TITLE
Spiral Linux Gnome Edition is AMAZING!

### DISCUSSION THREAD

**Discussion by v4ncouver** [SpiralLinux/SpiralLinux-project#323] [General]
Hello,
I have installed Gecko Linux in the past but always had issues with sound and other things so I stayed with openSUSE Tumbleweed. The other day, I installed SLDE and noticed right away that things a different. Everything works and I am planing to install your distro on my ASUS Zephyrus laptop too.

If you ever release a **minimal** version with just your modifications but no office and other optional apps which I won't use then please let me know. That would save me removing a ton of stuff.

Lastly, I read some review where someone complained about your wallpapers. Would you mind if I make some using your branding "Spiral Linux"?

**Comment by geckolinux** [comment]
Hi there, really glad to hear that it's working well for you, thanks for saying so!

> If you ever release a minimal version with just your modifications but no office and other optional apps which I won't use then please let me know. That would save me removing a ton of stuff.

It would be kind of a lot of work to generate 7 more minimal editions. You could maybe try the Builder edition, which comes with all the low-level stuff preconfigured, but it has no default configuration for whatever desktop you choose.

> Lastly, I read some review where someone complained about your wallpapers. Would you mind if I make some using your branding "Spiral Linux"?

;-) I really don't worry much about the wallpapers. That's always the first thing that users change, and individual user preferences always vary wildly, so I don't see much value in using time and disk space on the ISO to include more wallpapers. I'm sure that users already have online or offline sources of wallpapers that they can use if they care enough to change the default.

Thanks again for the kind words!
